src/EventStore.Client.UserManagement/EventStoreUserManagementClientCollectionExtensions.cs

```diff
@@ -27,6 +41,26 @@ public static class EventStoreUserManagementClientCollectionExtensions {
 				options.ConnectivitySettings.Address = address;
 				options.CreateHttpMessageHandler = createHttpMessageHandler;
 			});
+#endif
+
+#if NETCOREAPP3_1
+		/// <summary>
+		/// Adds an <see cref="EventStoreUserManagementClient"/> to the <see cref="IServiceCollection"/>.
+		/// </summary>
+		/// <param name="services"></param>
+		/// <param name="address"></param>
+		/// <param name="createHttpMessageHandler"></param>
+		/// <returns></returns>
+		/// <exception cref="ArgumentNullException"></exception>
+		[Obsolete]
+		public static IServiceCollection AddEventStoreUserManagementClient(this IServiceCollection services,
+			// ReSharper disable once MethodOverloadWithOptionalParameter
+			Uri address, Func<HttpMessageHandler>? createHttpMessageHandler = null)
+			=> services.AddEventStoreUserManagementClient(options => {
+				options.ConnectivitySettings.Address = address;
+			});
```

Comment:
@thefringeninja same here, is `options.CreateHttpMessageHandler = createHttpMessageHandler;` missing?

Reply:
same as above